phpstan-unsupported-baseline.neon

```diff
@@ -20,7 +20,18 @@ parameters:
 			count: 1
 			path: src/Contract/Operation/Unpackable.php
 
+		-
+			message: "#^Parameter \\#1 \\$callable of class loophp\\\\collection\\\\Iterator\\\\ClosureIterator constructor expects callable\\(\\.\\.\\.mixed\\)\\: iterable\\<int, string\\>, Closure\\(mixed, bool\\)\\: Generator\\<int, string, mixed, void\\> given\\.$#"
```

Comment:
This is the only part (The 2 new entries in this file) that I cannot fix. If you have any idea, just let me know.

Reply:
it's a bit difficult to tell at a glance, but leave this will me I will try to see if I can figure out the problem and fix it

Reply:
Ok, thanks! :)

Reply:
@drupol so after testing I realised this happens when passing a callable which takes multiple parameters, both to the ClosureIterator and the Collection object itself. Why? I'm not entirely sure but see the latest commit and the other comment for the changes I did and let me know if you think that works